HunterPie/Features/Patcher/PatchHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq;
+using HunterPie.Core.Address.Map;
+using HunterPie.Core.Domain.Memory;
+
+namespace HunterPie.Features.Patcher;
+
+internal static class PatchHelper
+{
+    /// <inheritdoc cref="CheckMemory(IMemory,long,string)"/>
+    /// <param name="memoryLocationName">the name of an Address existing in <see cref="AddressMap"/>.</param>
+    public static bool CheckMemory(IMemory memory, string memoryLocationName, string expectedDataName)
+    {
+        var memoryLocation = AddressMap.GetAbsolute(memoryLocationName);
+        return CheckMemory(memory, memoryLocation, expectedDataName);
+    }
+
+    /// <summary>
+    /// Checks whether the process memory sequence starting from the specified memory address
+    /// has the same data.
+    /// </summary>
+    /// <param name="memory">the memory to inspect.</param>
+    /// <param name="memoryLocation">starting memory address.</param>
+    /// <param name="expectedDataName">the name of a byte sequence (Instruction) existing in <see cref="AddressMap"/>.</param>
+    public static bool CheckMemory(IMemory memory, long memoryLocation, string expectedDataName)
```

Comment:
Could be implemented as an extension method of `IMemory`

```suggestion
    public static bool CheckMemory(this IMemory memory, long memoryLocation, string expectedDataName)
```

Reply:
Good idea! Also changed the name a bit.